### PR TITLE
Doc page for Heading component

### PR DESCRIPTION
**Pull request by Lisa159**

### CODE REVIEW

**Comment by aweell** [comment]
The latest changes for the heading component weren't merged in master from 4.2.0 (including the changes in the levels `font-size`)

The updated documentation can be found in https://github.com/dxc-technology/halstack-style-guide/tree/4.2.0/guidelines/components/heading

**Review by raquelarrojo — CHANGES_REQUESTED** [review]

File: website/pages/components/header/index.tsx [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
       </Head>
-      <HeaderUsagePage></HeaderUsagePage>
+      <HeadingUsagePage></HeadingUsagePage>
     </>
   );
 };
 
 Index.getLayout = function getLayout(page: ReactElement) {
-  return <HeaderPageLayout>{page}</HeaderPageLayout>;
+  return <HeadingPageLayout>{page}</HeadingPageLayout>;
```
[the remark this answers is not in the document]
Comment (Lisa159) [reply website/pages/components/header/index.tsx:18]: Missclick, I was changing heading and looks like I changed header. I will revert it.

[the remark this answers is not in the document]
Comment (Lisa159) [reply website/pages/components/header/index.tsx]: Done.

File: website/screens/components/heading/usage/HeadingUsagePage.tsx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+          There are several scenarios where the use of heading is just right
+          when we are talking about the title for a section, Heading of a table,
+          a definition of the elements that will appear below the title or an
+          introduction for a paragraph.{" "}
+        </DxcText>
+        <DxcText as="p">
+          Apart from that, many of the design system components are making use
+          of headings in their implementation (e.g., card, dialog, sidenav and
+          so on).{" "}
+        </DxcText>
```
[the remark this answers is not in the document]
Comment (Lisa159) [reply website/screens/components/heading/usage/HeadingUsagePage.tsx:38]: Done.

File: website/screens/components/heading/usage/HeadingUsagePage.tsx [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+      </DxcStack>
+      <DxcStack gutter="large">
+        <HeadingLink level={3}>Do&#39;s</HeadingLink>
```
[the remark this answers is not in the document]
Comment (Lisa159) [reply website/screens/components/heading/usage/HeadingUsagePage.tsx:39]: Done.

File: website/screens/components/heading/usage/HeadingUsagePage.tsx [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+        <DxcText as="p">
+          Use <Code>Heading</Code> components from <Code>H1</Code> to{" "}
+          <Code>H5</Code> when creating content hierarchy in the page.
+        </DxcText>
+        <DxcList>
```
[the remark this answers is not in the document]
Comment (Lisa159) [reply website/screens/components/heading/usage/HeadingUsagePage.tsx:40]: Done.

File: website/screens/components/heading/usage/HeadingUsagePage.tsx [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+          <DxcText>
+            Use headings for page headings and for specific sections of content.
+          </DxcText>
+          <DxcText>
+            Use purposeful words to summarize the content that follows.
+          </DxcText>
+        </DxcList>
+      </DxcStack>
+      <DxcStack gutter="large">
+        <HeadingLink level={3}>Dont&#39;s</HeadingLink>
```
[the remark this answers is not in the document]
Comment (Lisa159) [reply website/screens/components/heading/usage/HeadingUsagePage.tsx:52]: Done.

**Comment by Lisa159** [comment]
> The latest changes for the heading component weren't merged in master from 4.2.0 (including the changes in the levels `font-size`)
> 
> The updated documentation can be found in https://github.com/dxc-technology/halstack-style-guide/tree/4.2.0/guidelines/components/heading

@aweell Updated Heading with version 4.2.0.

**Review by raquelarrojo — APPROVED** [review]

**Pull request merged**